test/compilationTests/zeppelin/token/LimitedTransferToken.sol

```diff
@@ -51,7 +51,7 @@ contract LimitedTransferToken is ERC20 {
    * @dev Overwriting transferableTokens(address holder, uint64 time) is the way to provide the 
    * specific logic for limiting token transferability for a holder over time.
    */
-  function transferableTokens(address holder, uint64 time) view public returns (uint256) {
+  function transferableTokens(address holder, uint64 time) public returns (uint256) {
```

Comment:
Why is this non-view? `balanceOf` is still marked as `view`.

Reply:
Hm, is it because the `interface` is declared non-view and this still doesn't let a stricter set to be set than in the interface?

Reply:
Actually, at the moment I don't see a reason for this one and the same one in ``VestedToken.sol`` not both to be view - I'll change it and see what the tests say.